module-os/board/rt1051/include/FreeRTOSConfig.h

```diff
@@ -69,23 +67,15 @@ extern "C"
 #define configQUEUE_REGISTRY_SIZE     32
 #define configUSE_QUEUE_SETS          1
 #define configUSE_TIME_SLICING        1
-#ifdef TARGET_Linux
-#define configUSE_NEWLIB_REENTRANT 0
-#else
 #define configUSE_NEWLIB_REENTRANT 1
```

Comment:
With this version change, linux can use reentrant?

Reply:
FreeRTOS configuration files have been split, i.e. each board/platform has its configuration.